lib/inquisitor.ex

```diff
@@ -106,4 +113,15 @@ defmodule Inquisitor do
     |> List.last()
     |> Mix.Utils.underscore()
   end
+
+  @doc false
+  def handle_non_integer(i) do
```

Comment:
Thanks @marktran ! Should have been more careful copying that test over.  I'll fix shortly.